freqtrade/data/btanalysis.py

```diff
@@ -129,16 +129,21 @@ def load_trades_from_db(db_url: str) -> pd.DataFrame:
     return trades
 
 
-def load_trades(source: str, db_url: str, exportfilename: Path) -> pd.DataFrame:
+def load_trades(source: str, db_url: str, exportfilename: Path, no_trades: bool) -> pd.DataFrame:
```

Comment:
I'd default this to False - so it's not a required argument.

```suggestion
def load_trades(source: str, db_url: str, exportfilename: Path, no_trades: bool = False) -> pd.DataFrame:
```

Reply:
Thats nice!